org.jacoco.core/src/org/jacoco/core/internal/flow/MethodProbesVisitor.java

```diff
@@ -147,4 +148,8 @@ public void visitLookupSwitchInsnWithProbes(final Label dflt,
 			final int[] keys, final Label[] labels, final IFrame frame) {
 	}
 
+	public void accept(final MethodNode methodNode,
```

Comment:
"Nothing is more permanent than the temporary" :wink: but future will show. We kind of already had such mix - see `org.jacoco.core.internal.flow.LabelFlowAnalyzer#markLabels`, and exactly this gave me an idea of how to use ASM Tree API for filters without massive changes of everything, so this mix is worth it.